cmd/client/lib/cli.go

```diff
@@ -172,20 +179,14 @@ func buildGrpcClient(c *cli.Context, info **chain.Info) ([]client.Client, error)
 			}
 		}
 
-		if *info != nil && len(hash) == 0 {
-			hash = (*info).Hash()
+		if info != nil && len(hash) == 0 {
+			hash = info.Hash()
 		}
 
 		gc, err := grpc.New(c.String(GRPCConnectFlag.Name), c.String(CertFlag.Name), c.Bool(InsecureFlag.Name), hash)
 		if err != nil {
 			return nil, err
 		}
-		if *info == nil {
-			*info, err = gc.Info(context.Background())
-			if err != nil {
-				return nil, err
-			}
-		}
```

Comment:
Why the removal? 
Can info really never be `nil` now? 
Seems it could still be nil and needs to be populated, no?